src/client/components/DateInput/index.tsx

```diff
@@ -72,10 +65,17 @@ export const DateInput: React.FC<DateInputProps> = ({
   hasCurrentInsurer,
 }) => {
   const textKeys = useTextKeys()
+  const containerRef = useRef<HTMLDivElement>(null)
+
+  const handleClickOutside = useCallback(() => {
+    if (open) {
+      setOpen(false)
+    }
+  }, [open, setOpen])
```

Comment:
See other comment.

Reply:
Here's a little bit different because we receive `setOpen` as `prop`. If I remove it from dependency array the "missing dependency" lint warning will be shown. Which makes sense if we think on the scenario where consumer passes a different `setOpen` callback, right?!

Reply:
yeah, if it's props they should be in the dependency array, you are right :)